server/routes/detailsOfOffence/detailsOfOffence.ts

```diff
@@ -90,10 +83,11 @@ export default class DetailsOfOffencePage {
     }
     // Saving the offences for a draft just means continue
     if (!this.pageOptions.displaySessionData()) {
-      this.allOffencesSessionService.deleteSessionOffences(req, adjudicationNumber)
+      // this.allOffencesSessionService.deleteSessionOffences(req, adjudicationNumber)
```

Comment:
I guess this line just needs removing now?

Reply:
yeah, i cant find it locally though.  bizarre.  Will push again!